### ISSUE TITLE
Typescript-files with lodash

### ISSUE THREAD

**Issue by aleks-qq** [allenhwkim/ngentest#2]
Is there any plan on making gentest to work with typescript files that have lodash imported?
I'm currently getting: 
```
TypeError: Cannot read property 'map' of undefined
    at parsed.imports.forEach.mport (/usr/local/lib/node_modules/ngentest/src/lib/parse-typescript.js:27:39)
    at Array.forEach (<anonymous>)
    at parseTypescript (/usr/local/lib/node_modules/ngentest/src/lib/parse-typescript.js:26:18)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
    at Function.Module.runMain (module.js:678:11)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:608:3
```

I have lodash imported the following way:

```
import * as _ from 'lodash';
```

**Comment by aleks-qq** [comment]
Also I noticed that if the component doesn't have a constructor it will give the following error:
```
TypeError: Cannot read property 'forEach' of undefined
    at getDirectiveData (/usr/local/lib/node_modules/ngentest/src/get-directive-data.js:54:35)
    at parseTypescript.then.tsParsed (/usr/local/lib/node_modules/ngentest/index.js:33:17)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)
    at Function.Module.runMain (module.js:678:11)
    at startup (bootstrap_node.js:187:16)
    at bootstrap_node.js:608:3
```

**Comment by allenhwkim** [comment]
thanks. I will look into it.

**Comment by allenhwkim** [comment]
This has been fixed and published as version 0.2.1
Please check and close this issue. 

Thanks for reporting this issue again.

**Comment by aleks-qq** [comment]
Works perfectly, thanks
